docs/hub/_toctree.yml

```
@@ -29,6 +29,11 @@
       title: Carbon Emissions
     - local: models-gated
       title: Gated models
+    - local: model-card-guidebook
+      title: Model Card Guidebook
+      sections:
+      - local: model-card-annotated
+        title: Annotated Model Card
```

Hey @meg-huggingface :wave: The documentation is not building (see red :x: in github checks) because it's looking for a `model-card-guidebook.mdx` that does not exists (yet ?). I know this is only temporary work and will be completed with next chapters. What I advice you to do it to remove the "Model Card Guidebook" page for now since you don't have a content for it yet. 
This way documentation will build properly and you'll be able to see your changes. You'll be able to put back this page once you start to write it (=> basically the "home page" of a section is not mandatory in the docs).

```suggestion
    - title: Model Card Guidebook
      sections:
      - local: model-card-annotated
        title: Annotated Model Card
```

---

I actually made the change to make the docs work but we can revert it back if necessary.